index.html

```diff
@@ -10,188 +10,223 @@
+                        </ul>  
+                    </div>
+                    <div class="proj-btn">
+                        <button type="button" class="btn btn-primary">See Project</button>
+                    </div>
                 </div>
+                
             </div>
             <div class="proj-4">
                 <div class="proj-img">
-                    <img src="images/proj-img.png" alt="Project Screen Shot">
-                </div>
-                <div class="proj-info">
-                    <div class="proj-title">
-                        <h3>Multi-Post Stories</h3>
-                </div>
-                <div class="proj-desc">
-                    <p class="project-desc-p">
-                        A daily selection of privately personalized reads; no accounts or
-                        sign-ups required. has been the industry's standard dummy text ever
-                        since the 1500s, when an unknown printer took a standard dummy text.
-                      </p>
+                    <img src="images/proj-img.png" alt="Project Screen Shot" class="img-2-4">
                 </div>
+                <div class="poj-det">
+                    <div class="proj-info">
+                        <div class="proj-title">
+                            <h3>Multi-Post Stories</h3>
+                    </div>
+                    <div class="proj-desc">
+                        <p class="project-desc-p">
+                            A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. has been the industry's standard dummy text ever
+                            since the 1500s, when an unknown printer took a standard dummy text.
+                          </p>
+                    </div>
+                    </div>
+                    <div class="">
+                        <ul class="proj-tech">
+                            <li class="border-class">css</li>
+                            <li class="border-class">html</li>
+                            <li class="border-class">bootstrap</li>
+                            <li class="noborder">ruby</li>
+                        </ul>  
+                    </div>
+                    <div class="proj-btn">
+                        <button type="button" class="btn btn-primary">See Project</button>
+                    </div>
                 </div>
-                <div class="">
-                    <ul class="proj-tech">
-                        <li class="border-class">css</li>
-                        <li class="border-class">html</li>
-                        <li class="border-class">bootstrap</li>
-                        <li class="noborder">ruby</li>
-                    </ul>  
-                </div>
-                <div class="proj-btn">
-                    <button type="button" class="btn btn-primary">See Project</button>
-                </div>
+                
```

Comment:
- [ ] [OPTIONAL] I think it will be nice if you could please align your contact form as per the Figma design.

 Your Design           |
:------------------------:|
![e2](https://user-images.githubusercontent.com/39523312/149410550-71a06ed1-3fa3-4496-a9cd-1c383b39ec47.png)

 Figma Design           |
:------------------------:|
![e1](https://user-images.githubusercontent.com/39523312/149410561-eb8a03f1-026f-4fa0-9f55-c76cf14de6bc.png)